src/NotificationDbService.integration.js

```diff
@@ -0,0 +1,48 @@
+require('dotenv').config()
+const mongoose = require('mongoose')
+mongoose.promise = global.Promise
+
+const { getUsers } = require('./UserDbService')
+const { getLatestNewsArticle } = require('./newsDbService')
+const { saveNotifications, deleteNotification } = require('./NotificationDbService')
+
+describe('Notification Db Service ', () => {
+	let articles
+	let users
+	beforeAll(async () => {
+		mongoose.connect(process.env.DATABASE_URL, { useNewUrlParser: true })
```

Comment:
@shrijan-siris have you got a chance to try this?
https://dev.to/paulasantamaria/testing-node-js-mongoose-with-an-in-memory-database-32np

Reply:
seems great